Dockerfile.debian_package

```diff
@@ -2,20 +2,20 @@ FROM debian:9
 
 # https://dotnet.microsoft.com/download/linux-package-manager/debian9/sdk-current
 RUN apt-get update \
- && apt-get install -y apt-transport-https debhelper gnupg wget \
+ && apt-get install -y apt-transport-https debhelper gnupg wget devscripts \
  && wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg \
  && mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/ \
  && wget -q https://packages.microsoft.com/config/debian/9/prod.list \
  && mv prod.list /etc/apt/sources.list.d/microsoft-prod.list \
  && chown root:root /etc/apt/trusted.gpg.d/microsoft.asc.gpg \
  && chown root:root /etc/apt/sources.list.d/microsoft-prod.list \
- && apt-get update \
- && apt-get install -y dotnet-sdk-2.2
+ && apt-get update
 
 WORKDIR /repo
 COPY . .
 
-RUN dpkg-buildpackage -us -uc \
+RUN yes|mk-build-deps -i \
```

Comment:
mk-build-scripts doesn't offer a "-y" option, so I have to pipe `yes` into it to get it to run noninteractively.